azuredeploy.json

```diff
@@ -86,15 +86,77 @@
             }
         }
     },
-    "variables": {
+    "variables": {        
+        "functionActions": [
+            "Microsoft.Storage/storageAccounts/listKeys/action",
+            "Microsoft.Storage/storageAccounts/blobServices/containers/read",
+            "Microsoft.Storage/storageAccounts/blobServices/generateUserDelegationKey/action",
+            "Microsoft.Web/sites/config/list/action",
+            "Microsoft.Web/sites/config/write",
+            "Microsoft.Web/sites/functions/read"
+        ],
+        "roleDefinitionName": "[guid(subscription().id, string(variables('functionActions')), uniqueString(resourceGroup().id))]",
+        "functionRoleName": "[format('fingerprint-pro-function-app-role-{0}', uniqueString(resourceGroup().id))]",
+        "userAssignedIdentityName": "[concat('fingerprint-pro-ua-', uniqueString(resourceGroup().id))]",
+        "roleAssignmentName": "[guid(concat(variables('userAssignedIdentityName'), resourceGroup().id, 'contributor'))]",
         "functionAppName": "[format('{0}{1}', parameters('functionAppName'), uniqueString(resourceGroup().id))]",
         "hostingPlanName": "[variables('functionAppName')]",
-        "appInsightsName": "[format('AppInsights -{0}', uniqueString(resourceGroup().id))]",
+        "appInsightsName": "[format('AppInsights-{0}', uniqueString(resourceGroup().id))]",
         "storageAccountName": "[format('fnstor{0}', uniqueString(resourceGroup().id))]",
+        "containerName": "function-releases",
         "functionNameComputed": "fingerprint-pro-azure-integration",
-        "functionRuntime": "node"
+        "functionRuntime": "node",
+        "packageZipUri": "https://github.com/fingerprintjs/fingerprint-pro-azure-integration/releases/latest/download/package.zip",
+        "packageName": "package.zip",
+        "copyPackageScriptName": "copyPackage",
+        "$fxv#0": "# copy the file to the local disk\r\nInvoke-WebRequest -Uri \"${env:packageZipUri}\" -OutFile \"${env:packageName}\"\r\n\r\n# get storageAccount context using the key\r\n$ctx = New-AzStorageContext -StorageAccountName \"${Env:storageAccountName}\" -StorageAccountKey \"${Env:storageAccountKey}\"\r\n\r\nNew-AzStorageContainer -Context $ctx -Name \"${env:containerName}\" -Verbose\r\n\r\n# copy blob\r\nSet-AzStorageBlobContent -Context $ctx `\r\n                         -Container \"${Env:containerName}\" `\r\n                         -Blob \"${env:packageName}\" `\r\n                         -StandardBlobTier 'Hot' `\r\n                         -File \"${env:packageName}\"\r\n"
```

Comment:
It is possible somehow to make this more readable? I'm worried that it might be confusing for customers that will want to use this integration

Reply:
Got it, let's add a mention in docs about this script then